### ISSUE TITLE
wrong alpha value 

### ISSUE THREAD

**Issue by guo-yong-zhi** [JuliaIO/ImageIO.jl#40]
```julia
]activate imgio
]add ImageIO
using FileIO
@show load("./heart_mask.png")[300,300]
@show load("./heart_mask.png")[300,300]
```
->
>(load("./heart_mask.png"))[300, 300] = RGBA{N0f8}(1.0,0.753,0.796,0.941)
(load("./heart_mask.png"))[300, 300] = RGBA{N0f8}(1.0,0.753,0.796,0.753)

The image can be found here: https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png

>versioninfo:
Julia Version 1.7.0
Commit 3bf9d17731 (2021-11-30 12:12 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, skylake)

>Status `~/imgio/Project.toml`
[82e4d734] ImageIO v0.5.9

**Comment by Drvi** [comment]
This should be fixed in the `0.3.13` version of PNGFiles.jl.

```
julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)

julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)

julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)

julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)

julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)

julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)

julia> PNGFiles.load(Downloads.download("https://raw.githubusercontent.com/guo-yong-zhi/WordCloud.jl/master/res/heart_mask.png"))[300, 300]
RGBA{N0f8}(1.0,0.753,0.796,1.0)
```

**Comment by johnnychen94** [comment]
Can we close this?
